### PR TITLE
GitHub source links

### PR DESCRIPTION
**Pull request by danielnaab**
Adds Github source links to `sch:assert`s, `xspec:expectAssert`s, and `xspec:expectNotAssert`s. See: #625 

This PR also makes a couple of the CLI build steps more consistent with each other in how they are parameterized.

### CODE REVIEW

**Review by GaryGapinski — APPROVED** [review]
Looks OK; some minor comments.

File: src/web/src/browser/presenter/index.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
+import type { GetSchematronAssertions } from '@asap/shared/domain/schematron';
 import type { AnnotateXMLUseCase } from '@asap/shared/use-cases/annotate-xml';
-import type { GetXSpecScenarioSummaries } from '@asap/shared/use-cases/assertion-documentation';
 import type { GetAssertionViews } from '@asap/shared/use-cases/assertion-views';
 import type { OscalService } from '@asap/shared/use-cases/oscal';
-import type { GetSchematronAssertions } from '@asap/shared/use-cases/schematron';
+import type { GetXSpecScenarioSummaries } from '@asap/shared/use-cases/xspec-summary';
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply src/web/src/browser/presenter/index.ts:5]: Good question..

File: src/web/src/shared/domain/text.test.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+import { linesOf, linesOfXml } from './text';
+
+describe('text domain', () => {
+  describe('linesOf', () => {
+    it('returns line numbers for sch:assert', () => {
+      const lineNumbers = linesOf(
+        TEST_SCH,
+        `<sch:assert
+                diagnostics="has-web-applications-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.5"
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply src/web/src/shared/domain/text.test.ts:11]: This is just mock test data, so could be any arbitrary string.

File: src/web/src/browser/views/styles/ValidatorReport.scss [hunk 3]
```diff
@@ -16,9 +16,9 @@ details > summary {
...
 
 details > summary .learn-more::after {
-  content: '\22C1';
+  content: ' \22C1';
 }
 
 details[open] > summary .learn-more::after {
-  content: '\22C0';
+  content: ' \22C0';
 }
```
[the remark this answers is not in the document]
Comment (danielnaab) [reply src/web/src/browser/views/styles/ValidatorReport.scss:24]: No, they're just being used as arrows. @fpigeonjr do screenreaders do anything with this content?

[the remark this answers is not in the document]
Comment (fpigeonjr) [reply src/web/src/browser/views/styles/ValidatorReport.scss:24]: > No, they're just being used as arrows. @fpigeonjr do screenreaders do anything with this content?

@danielnaab I'm not sure. I'll run it through VoiceOver today and let you know.

[the remark this answers is not in the document]
Comment (GaryGapinski) [reply src/web/src/browser/views/styles/ValidatorReport.scss:24]: https://unicode-table.com/en/sets/arrow-symbols/ has a compendium.

[the remark this answers is not in the document]
Comment (fpigeonjr) [reply src/web/src/browser/views/styles/ValidatorReport.scss:24]: VoiceOver in Safari does read the unicode symbol and vertical lines. 

Screen Shot 2022-08-25 at 1 24 49 PM

**Pull request merged**